VehicleMsg[] lane_vehicles
LanePointMsg[] lane_points
int64 lane_id
PedestrainMsg[] crossing_pedestrain
geometry_msgs/Pose2D origin_global_pose
float64 ego_offset
bool left_turning_lane
bool right_turning_lane
bool right_most_lane